lib/cli/local/index.js

```diff
@@ -67,7 +67,7 @@ export const builder = function(yargs) {
             'knows this token can freely enter the system if they want, so keep ' +
             'it safe.'
             , type: 'string'
-            , default: 'kute kittykat'
+            , default: process.env.SECRET || 'secret'
```

Comment:
<img width="802" alt="image" src="https://github.com/user-attachments/assets/9da48cb3-d8bc-497b-a54f-02c1ba51ee6c" />
Four usages left of 'kute kittykat'